hexa/workspaces/models.py

```diff
@@ -55,6 +56,7 @@ def create_if_has_perm(
         if description is None:
             create_kwargs["description"] = "This is a workspace for {}".format(name)
 
+        create_database(create_kwargs["slug"])
```

Comment:
The slug as-is is not valid as a database name. You need to prepare the slug to match the required format.

I'd also store on the workspace model the db_name & db_password